tests/Rector/v13/v0/RemoveMmHasUidFieldRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v13\v0\RemoveMmHasUidFieldRector\Fixture;
+
+return [
+    'ctrl' => [],
```

Comment:
Try it without ctrl for single TCA column definitions in Overrides

Reply:
We add a new fixture with the TCA globals example. It works, but the control is mandatory.